nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -61,6 +73,7 @@ let
       #Let's hope 15M is enough
       dd bs=2048 count=7680 if=/dev/zero of="$out"
```

Comment:
Yes, but that would mean all ISO images will increase in size quite a bit. Ideal would be to measure how much is needed and create an image that fits those needs. Not sure if `du` is reliable for that (now much does FAT use?), but it's a good step in the right direction I think.

Also, I've created #6796, so that we have a better place for this conversation. (I have trouble finding these in-code conversations each time ;)).